### PR TITLE
Ajay tripathy upgrade prom thanos

### PR DESCRIPTION
**Pull request by AjayTripathy**
Bump prometheus and thanos-store versions

### CODE REVIEW

File: cost-analyzer/values-thanos.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ prometheus:
     enableAdminApi: true
     sidecarContainers:
     - name: thanos-sidecar
-      image: thanosio/thanos:v0.15.0
+      image: thanosio/thanos:v0.19.0
```
Comment (mbolt35) [inline cost-analyzer/values-thanos.yaml:31]: Why just the sidecar here?

Comment (mbolt35) [reply cost-analyzer/values-thanos.yaml:31]: Ok, nevermind - I see that sidecar didn't get bumped in the previous update.

Comment (AjayTripathy) [reply cost-analyzer/values-thanos.yaml:31]: Yes. Two reasons:

Sidecar interoperates with prometheus closely, so prometheus being significantly behind sidecar seemed bad

Users don't upgrade all clusters simultaneously anyway, so we needed to support some level of interoperability anyway between 0.15 and 0.19.

**Review by mbolt35 — APPROVED** [review]
LGTM

**Pull request merged**